rewrite-java/build.gradle.kts

```diff
@@ -20,6 +20,7 @@ dependencies {
     implementation("org.antlr:antlr4:4.8-1")
 
     implementation("commons-lang:commons-lang:latest.release")
+    implementation("org.apache.commons:commons-text:latest.release")
```

Comment:
```suggestion
```
This doesn't seem to actually be used anywhere. Let's not take new dependencies without good reason,

Reply:
Removed, thank you!